build.gradle

```diff
@@ -114,8 +114,8 @@ dependencies {
     compile group: 'com.itextpdf', name: 'itext7-core', version: '7.1.3', ext: 'pom'
     compile group: 'com.itextpdf', name: 'html2pdf', version: '2.1.0'
     compile("org.springframework.boot:spring-boot-starter-thymeleaf")
-    compile group: 'org.apache.xmlgraphics', name: 'batik-codec', version: '1.9.1'
-    compile group: 'org.apache.xmlgraphics', name: 'batik-transcoder', version: '1.9'
+    compile group: 'org.apache.xmlgraphics', name: 'batik-codec', version: '1.11'
```

Comment:
I bumped to the latest versions of the image libraries we use in IMS for svg -> png